sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -56,4 +72,43 @@ export class ApplicationAESTController extends BaseController {
       application,
     );
   }
+
+  /**
+   * Creates a new MSFAA number to be associated with the student, cancelling any
+   * pending MSFAA for the particular offering intensity and also associating the
+   * new MSFAA number to any pending disbursement for the same offering intensity.
+   * @param applicationId reference application id.
+   * @returns the newly created MSFAA.
+   */
+  @Post(":applicationId/reissue-msfaa")
```

Comment:
While reissuing msfaa, wouldn't it be better to have the student as part of URI as we issue/re-issue msfaa for a student even though it happens on an application.?

Reply:
Honestly, I do not see any benefit, feel free to clarify if needed.
The MSFAA is issued for a student because of an application. Having the application and a student as a separate parameters opens the possibility of even providing a student from a different application.

Reply:
I was thinking back and forth but what you said makes sense.